db/migrate/20201014171538_add_journalist_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddJournalistToArticles < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :journalist_id, :integer
```

Comment:
please refactor this to make sure it works as intended

`add_reference :articles, :journalist, foreign_key: { to_table: :users }`

Reply:
We updated it!